### DISCUSSION TITLE
Getting 'File could not be loaded. Please check your internet connection.' when editing .md files

### DISCUSSION THREAD

**Discussion by iGadget** [nextcloud/all-in-one#4294] [Questions] [answered]
As per the title, I'm getting this error randomly when editing .md files in the browser (both via Files and Collectives app), together with a nice blue button 'Reconnect'.
When I click this 'Reconnect' button it usually still fails to connect and I have to manually reload the entire page to restore the connection.
I tried applying the solution mentioned [here](https://github.com/nextcloud/all-in-one/discussions/2557), but the issue remains. 

Now I can imagine this problem actually runs deeper (i.e. DNS, network) and I only notice it explicitly when editing the .md files. 
If this is the case, where should I start digging and which route should I follow to get to the root cause?

**Comment by szaimen** [comment, marked as answer]
Hi, this sounds like a bug in the text app. Feel free to create an issue here: https://github.com/nextcloud/text/issues
